src/components/Pages/Landing/UseCasesList/UseCasesList.module.css

```diff
@@ -0,0 +1,82 @@
+.useCasesList {
+  padding-bottom: var(--m-4);
+  background-color: var(--color-white);
+}
+
+.title {
+  margin-bottom: var(--m-3);
+  font-size: 26px;
+
+  @media (--md-scr) {
+    font-size: var(--lp-header-2);
+  }
+}
+
+.container {
+  max-width: 1400px;
+  margin: 0 auto;
+}
+
+.items {
+  list-style: none;
+  padding: 0;
+  margin: 0;
+  display: grid;
+  gap: var(--m-2);
+  grid-template-columns: minmax(0, 1fr);
+
+  @media (--md-scr) {
+    grid-template-columns: repeat(2, minmax(0, 1fr));
+  }
+
+  @media (--lg-scr) {
+    grid-template-columns: repeat(3, minmax(0, 1fr));
+  }
+
+  li {
+    margin-top: 0 !important;
+  }
+}
+
+.item {
+  display: block;
+  padding: var(--m-2) var(--m-3);
+  border-radius: 12px;
+  border: 1px solid var(--color-tonal-neutral-1);
+  transition: all 0.2s ease-in-out;
+  text-decoration: none;
+  height: 100%;
+
+  h3 {
+    margin: 0 !important;
+    font-weight: var(--fw-bold);
+    line-height: 1.5;
+    color: var(--color-black);
+    font-size: var(--fs-text-xl);
+    transition: color 0.2s ease-in-out;
+
+    @media (--md-scr) {
+      font-size: var(--fs-header-4);
+    }
+  }
+
+  p {
+    color: var(--color-foreground-muted);
+    margin: 0;
+    line-height: 1.625;
+    font-size: var(--fs-text-md);
+
+    @media (--md-scr) {
+      font-size: var(--fs-text-lg);
+    }
+  }
+}
+
+a.item:hover {
+  text-decoration: none;
+  background: var(--color-white);
+  box-shadow: 0 4px 12px rgba(81, 47, 201, 0.15);
```

Comment:
Let's kill the border on hover, too, with a `border-color: rgba(0,0,0,0)` style so that it fades out as the shadow fades in.

We should also add a task to standardize this shadow in a variable. so we don't keep repeating it across components.